_opensearch/java-rest-high-level.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Java high-level REST client
+nav_order: 97
+---
+
+# Java high-level REST client
+
+The Elasticsearch OSS Java high-level REST client allows you to interact with your OpenSearch clusters and indices through request objects and response objects. To create an index, add data to a document, or complete some other operation, you submit request objects to your cluster, and you get back response objects that have all of the available information, such as the associated index or ID, returned from your cluster.
+
+## Setup
+
+To start using the Elasticsearch OSS Java high-level REST client, ensure that you have the following dependency in your project's `pom.xml` file:
+
+```
+<dependency>
+  <groupId>org.elasticsearch.client</groupId>
+  <artifactId>elasticsearch-rest-high-level-client</artifactId>
+  <version>7.10.2</version>
+</dependency>
+```
+
+You can now start your OpenSearch cluster. Any version of OpenSearch works with the Elasticsearch OSS high-level REST client.
```

Comment:
So while this is true, we probably want to say something like, "The 7.10.2 client works with the 1.x versions of OpenSearch."